src/applications/pre-need/config/form.jsx

```diff
@@ -68,6 +68,12 @@ const {
 
 const nonRequiredFullName = _.omit('required', fullName);
 
+function currentlyBuriedPersonsMinItem() {
```

Comment:
You could also do this with one `_.merge` statement, but this function is correct and I don't have a strong preference either way.